functional/MBCS_Tests/i18n/playlist.xml

```diff
@@ -150,6 +157,13 @@ limitations under the License.
 	</test>
 	<test>
 		<testCaseName>MBCS_Tests_i18n_zh_CN_aix</testCaseName>
+		<disables>
+			<disable>
+				<comment>https://github.com/eclipse-openj9/openj9/issues/15249</comment>
+				<impl>openj9</impl>
+				<version>19+</version>
+			</disable>
+		</disables>
```

Comment:
Just double-check, in addition to the above exclusions, do we need to exclude MBCS_Tests_i18n_Zh_CN_aix and MBCS_Tests_i18n_ZH_CN_aix below?

Reply:
No need for such excludes since there was no failure reported for `MBCS_Tests_i18n_Zh_CN_aix_0` and `MBCS_Tests_i18n_ZH_CN_aix_0` at `job/Test_openjdknext_j9_special.functional_ppc64_aix_Personal/1/tapResults/`.